### PR TITLE
chore: remove snapshots from component packages

### LINKED ISSUE #962: Remove snapshots from component libraries
Related proposal #956 

The packages `fast-components-react-base` and `fast-components-react-msft` should have their dependency of the snapshots removed and all snapshots replaced with the appropriate unit testing.

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>
closes #962 
<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-react-msft/src/heading/heading.spec.tsx [hunk 1]
```diff
@@ -56,9 +48,30 @@ describe("heading", (): void => {
         expect(rendered.find(handledProps.tag).prop("aria-hidden")).toEqual(true);
     });
 
+    test("should render a default `tag` when no `tag` prop is passed", () => {
+        const rendered: any = mount(<MSFTHeading />);
+
+        expect(rendered.exists(HeadingTag.h1)).toBe(true);
+    });
+
     test("should render the correct `tag` when `tag` prop is passed", () => {
```
Comment (janechu) [inline packages/fast-components-react-msft/src/heading/heading.spec.tsx]: Can we have a test to render each valid tag

File: packages/fast-components-react-msft/src/heading/heading.spec.tsx [hunk 2]
```diff
@@ -56,9 +48,30 @@ describe("heading", (): void => {
...
         const rendered: any = mount(<MSFTHeading tag={HeadingTag.h3} />);
 
         expect(rendered.exists(HeadingTag.h3)).toBe(true);
     });
+
+    test("should render the correct `size` when `size` prop is passed", () => {
```
Comment (janechu) [inline packages/fast-components-react-msft/src/heading/heading.spec.tsx]: Can we have a test to render each valid size

File: packages/fast-components-react-msft/src/paragraph/paragraph.spec.tsx [hunk 3]
```diff
@@ -66,4 +58,11 @@ describe("paragraph", (): void => {
 
         expect(rendered.prop("size")).toBe(ParagraphSize._3);
     });
+
+    test("should accept and render children", () => {
+        const rendered: any = shallow(<MSFTParagraph>Children</MSFTParagraph>);
+
+        expect(rendered.prop("children")).not.toBe(undefined);
```
Comment (nicholasrice) [inline packages/fast-components-react-msft/src/paragraph/paragraph.spec.tsx]: minor nit, but you don't really need both of these

Comment (chrisdholt) [reply packages/fast-components-react-msft/src/paragraph/paragraph.spec.tsx]: Will remove

File: packages/fast-components-react-msft/src/heading/heading.spec.tsx (hunk 2 above)
Comment (chrisdholt) [reply packages/fast-components-react-msft/src/heading/heading.spec.tsx]: Working on adding this and others

**Comment by nalogiudice** [comment]
Are we waiting on this for a reason? Quite old now.

**Pull request merged**